test/integration/targets/azure_rm_securitygroup/tasks/main.yml

```diff
@@ -102,6 +102,7 @@
       tags:
           testing: testing
           delete: on-exit
+      append_tags: false
```

Comment:
This should be false by default, since we need to keep currently behavior

Reply:
You're right, indeed. The Azure tag doc fragment needs to be updated then, since `append_tags` parameter is described as `True` by default.
I will add those changes to my PR.